File: 485_Max_Consecutive_Ones/solution.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_count, count = 0, 0
        for num in nums:
            count = count + 1 if num == 1 else 0
            max_count = max(max_count, count)
        return max_count

File: 485_Max_Consecutive_Ones/solution.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int max = 0, count = 0;
    for (int i = 0; i < numsSize; i++) {
        count = nums[i] == 1 ? count + 1 : 0;
        if (count > max) max = count;
    }
    return max;
}